creating-google-service-account.md

```diff
@@ -43,3 +57,66 @@ If you’d like to submit your Android app to Google Play Store with `eas submit
 [<img src="./assets/creating-google-service-account/11-set-permissions.png" width="800" />](./assets/creating-google-service-account/11-account-permissions.png)
 
 9. That's all! From now on, you can use the generated Google Service Account key to upload your app with `eas submit`.
+
+## Using a Google Service Account Key for Sending Android Notifications via FCM V1
+ 
+ Here are the steps to set up a Google Service Account Key for sending Android Notifications via FCM V1.
+
+### Configuring a new Google Service Account Key for Android Notifications using FCM V1
+
+Here are the steps to configure a new Google Service Account Key in EAS for sending Android Notifications using FCM V1.
+
+1. (If you already have a Firebase project for your app, continue to the next step.) Create a new Firebase project for your app in the [Firebase Console](https://console.firebase.google.com)
+
+[<img src="./assets/creating-google-service-account/01-new-google-cloud-project.png" width="800" />](./assets/creating-google-service-account/01-new-google-cloud-project.png)
+
+2. In the Firebase console, open Settings > [Service Accounts](https://console.firebase.google.com/project/_/settings/serviceaccounts/adminsdk) for your project
+
+[<img src="./assets/creating-google-service-account/02-service-accounts.png" width="800" />](./assets/creating-google-service-account/02-service-accounts.png)
+
+3. Click **Generate New Private Key**, then confirm by clicking **Generate Key**
+
+[<img src="./assets/creating-google-service-account/03-create-service-account.png" width="800" />](./assets/creating-google-service-account/03-create-service-account.png)
+
+4. Securely store the JSON file containing the private key
+
+[<img src="./assets/creating-google-service-account/04-manage-keys.png" width="800" />](./assets/creating-google-service-account/04-manage-keys.png)
+[<img src="./assets/creating-google-service-account/05-create-new-key.png" width="800" />](./assets/creating-google-service-account/05-create-new-key.png)
+[<img src="./assets/creating-google-service-account/06-key-type.png" width="800" />](./assets/creating-google-service-account/06-key-type.png)
+[<img src="./assets/creating-google-service-account/07-key-saved.png" width="800" />](./assets/creating-google-service-account/07-key-saved.png)
```

Comment:
These existing assets don't look correct in relation to the instructions. 

IIUC, if you have an existing service account for FCMV1 or dealing with the Google Play store submissions, you'd go through the GCP service account workflow. The pictures we currently have in `assets/creating-google-service-account/n.png` are for this GCP workflow. 

But for creating a new service key for FCMV1, we'll need to go through the Firebase console (which is basically a shim over GCP), but that flow will automatically create a service account key with the necessary permissions. So I'd expect the pictures to be from the Firebase console and look something like this:

<img width="734" alt="Screenshot 2024-02-07 at 12 36 15 PM" src="https://github.com/expo/fyi/assets/6380927/105c7fe7-6c1c-4979-9e8d-b9f27c84dec0">

Reply:
@quinlanj I've updated the assets, though I still need to fix the "Use existing service account key" flow (specifically the step where you grant the `Firebase Cloud Messaging API Admin` role to the existing service account).